Fix recommended import styles for js, ts, jsx and tsx files (#747)
withastro/docs: src/pages/en/guides/typescript.md

```

:::note

> `astro check` only checks types within `.astro` files, and `tsc --noEmit` only checks types within `.ts` and `.tsx` files. To check types within Svelte and Vue files, the [`svelte-check`](https://www.npmjs.com/package/svelte-check) and the [`vue-tsc`](https://www.npmjs.com/package/vue-tsc) packages may be used respectively

Author: Nit: period at the end of the sentence and slight word re-ordering to make “respectively” a little smoother.

```suggestion
`astro check` only checks types within `.astro` files, and `tsc --noEmit` only checks types within `.ts` and `.tsx` files. To check types within Svelte and Vue files, you can use the [`svelte-check`](https://www.npmjs.com/package/svelte-check) and the [`vue-tsc`](https://www.npmjs.com/package/vue-tsc) packages respectively.
```
